src/ui/default-prop-types.js

```diff
@@ -173,3 +177,44 @@ export const formItemLayoutShape = PropTypes.shape({
     sm: formItemDimensionShape.isRequired
   }).isRequired
 });
+
+export const importTaskShape = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  batchId: PropTypes.string.isRequired,
+  taskType: PropTypes.oneOf(Object.values(TASK_TYPE)),
+  processed: true,
+  attempts: [
+    {
+      startedOn: PropTypes.string,
+      completedOn: PropTypes.string,
+      errors: PropTypes.arrayOf(PropTypes.string).isRequired
+    }
+  ],
+  taskParams: {
+    key: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    slug: PropTypes.string.isRequired,
+    language: PropTypes.string.isRequired,
+    updatedOn: PropTypes.string,
+    importedRevision: PropTypes.string,
+    importableRevision: PropTypes.string.isRequired,
+    importType: PropTypes.oneOf(Object.values(DOCUMENT_IMPORT_TYPE))
+  }
+});
+
+export const importBatchShape = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  createdBy: userInDocShape.isRequired,
+  createdOn: PropTypes.string.isRequired,
+  completedOn: PropTypes.string,
+  batchType: PropTypes.oneOf(Object.values(BATCH_TYPE)),
+  batchParams: PropTypes.shape({
+    ...importSourceShape
+  }),
+  errors: PropTypes.arrayOf(PropTypes.any).isRequired
+});
+
+export const importBatchDetailsShape = PropTypes.shape({
+  ...importBatchShape,
```

Comment:
Same issue here (see my comment above)

Reply:
Solved both of them, that took a while to figure out :).